ws-android/Dockerfile

```diff
@@ -5,18 +5,29 @@ FROM cloud9/workspace
 # https://github.com/lukin0110/docker-android-build
 MAINTAINER Jeremy Ellis <keyfreemusic@gmail.com>
 
-ADD ./files/workspace /home/ubuntu/workspace
+# -------------------------- Transfer from Cloud9 Github to Docker -----------------------------------------
+# The Github is at https://github.com/c9/templates/tree/master/ws-android
+
+ADD ./files /home/ubuntu
+ADD ./files/profile.d-android.sh /etc/profile.d/android.sh
+
+# These commands just let us test the enviornment before submitting to cloud 9    
+ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
```

Comment:
I don't see us using the variable anywhere, and it's already specified in the profile.d file included above, so it's redundant and we now have to maintain it in two places.